src/App.tsx

```diff
@@ -20,17 +26,135 @@ class App extends React.Component<{}, State> {
       './img/9.png',
       './img/10.png',
     ],
+    step: 3,
+    frameSize: 3,
+    itemWidth: 130,
+    animationDuration: 1000,
+    infinite: false,
+  };
+
+  handleEvent = (event: ChangeEvent) => {
+    const { name, value } = event.target as HTMLInputElement;
+
+    switch (name) {
+      case 'step':
+        if (+value >= 0 && +value <= 10) {
+          this.setState({ [name]: +value });
+        }
+
+        break;
+
+      case 'frameSize':
+        if (+value >= 0 && +value <= 10) {
+          this.setState({ [name]: +value });
+        }
+
+        break;
+
+      case 'itemWidth':
+        if (+value >= 0 && +value <= 400) {
+          this.setState({ [name]: +value });
+        }
+
+        break;
+
+      case 'animationDuration':
+        if (+value >= 0 && +value <= 10000) {
+          this.setState({ [name]: +value });
+        }
+
+        break;
+
+      case 'infinite':
+        this.setState(state => ({
+          [name]: !state.infinite,
+        }));
```

Comment:
Rewrite it with if statement and min  max value sets to input
```suggestion
      if (name === 'infinite') {
        this.setState(state => ({
          [name]: !state.infinite,
        }));
      } else {
        this.setState({ [name]: +value });
      }
```